LunaMary/src/Util.java

```diff
@@ -0,0 +1,22 @@
+public class Util {
+
+
+    public Util() {
+
+    }
+
+    public Boolean validateSexo(String data) {
+        boolean textValid = false;
+
+        if (!data.isEmpty() && data.replace(" ", "").length() == 1 && textValid == false) {
+
+            textValid = true;
+        } else {
+
+            textValid = false;
```

Comment:
textValid is already false when you get to this point you could remove the else statement.